### DISCUSSION TITLE
File links with line number

### DISCUSSION THREAD

**Discussion by dpkpnm** [jakewvincent/mkdnflow.nvim#115] [Q&A]
How can you add file links with line number?

`[fileLink](/dir/filename:200)`

Adding this takes you to filename:200.md instead of going to a specific line.

**Comment by jakewvincent** [comment]
Hi there--yes, I haven't built in this kind of functionality, but it wouldn't be difficult to implement. Is this something that is normally built in to markdown compilers like Pandoc, that a link is generated that will go to whichever line corresponds to the line in the source markdown doc? Or is this built into Vimwiki, possibly?
